packages/edit-widgets/src/style.scss

```diff
@@ -1,3 +1,5 @@
+@import "../../interface/src/style.scss";
```

Comment:
I'm not sure how this worked before given that we didn't have explicit import from `block-editor` package either. 🤔